csharp/Ice/CustomError/README.md

```diff
@@ -0,0 +1,27 @@
+# Custom error
+
+The Custom error example shows how to define an operation that throws an exception defined as
+`GreeterException` when an issue is present.
+
+In this example, `GreeterException` holds `reason` and `error`, values defined as a string and a
+`GreeterError` meant to hold an error message and an error ID.
```

Comment:
Personally, I would remove this sentence.
Not because it's wrong, but because it's discussing the internals of `GreeterException`, which (to me) isn't a critical part of the demo. What's important is that you define an exception in Slice, and can throw it.
What information that exception happens to hold is secondary, and will vary.

To be clear, I'm **not** saying that this shouldn't be documented _anywhere_.
It's documented inside the Slice file itself, and that is a very good thing. I just don't think it's critical enough that we should explain it a _second time_ in the main README here. Ya feel?